app/src/common/shared/com/igalia/wolvic/ui/adapters/ThumbnailTask.java

```diff
@@ -66,11 +68,7 @@ protected Bitmap doInBackground(Void... voids) {
         }
     }
 
-    @Override
-    @Deprecated
```

Comment:
The problem I see here is that the Callable cannot be cancelled right? We should be able to still cancel file downloads/uploads

Reply:
Actually I'm not sure if `onCancelled()` method is ever used (I don't see it getting called anywhere).

And, this is about generating the preview thumbnails for the successfully downloaded files (https://github.com/Igalia/wolvic/pull/883/files#diff-04953cdc2a10f98cce3160405763d1420d889862807b36165f40810238c4d1d4R158) /files to upload, it's not about canceling file downloads/uploads.